root_VS2010/files/resource/Sql/sqlserver/Workflow/SQL/Table/M_User_Insert(sample).sql

```diff
@@ -1,24 +1,24 @@
-﻿USE [Workflow]
+USE [Workflow]
 GO
 
 INSERT INTO [dbo].[M_User]([Id], [Section], [Name], [PositionTitlesId])
     VALUES
-	(1   , 'AAAA' , 'A御中'   , 1 ),
+	(1   , 'AAAA' , 'A�䒆'   , 1 ),
```

Comment:
@daisukenishino 

The text is garbled.
Maybe the encoding of this file is Shift-JIS.